docker/config_template.yaml

```diff
@@ -44,11 +44,11 @@ persistence:
         default:
             sql:
                 pluginName: "{{ $db }}"
-                databaseName: "{{ default .Env.DBNAME "temporal" }}"
-                connectAddr: "{{ default .Env.MYSQL_SEEDS "" }}:{{ default .Env.DB_PORT "3306" }}"
+                databaseName: "{{ default (or .Env.DB_NAME .Env.DBNAME) "temporal") }}"
+                connectAddr: "{{ default (or .Env.DB_HOST .Env.MYSQL_SEEDS) "" }}:{{ default .Env.DB_PORT "3306" }}"
                 connectProtocol: "tcp"
-                user: "{{ default .Env.MYSQL_USER "" }}"
-                password: "{{ default .Env.MYSQL_PWD "" }}"
+                user: "{{ default (or .Env.DB_USER .Env.MYSQL_USER) "" }}"
```

Comment:
No need for MySQL specific variable names here. Using shared names where possible makes configuration more straight forward.